### DISCUSSION TITLE
Afficher nombre en cours de ressource en alarme d'une zone

### DISCUSSION THREAD

**Discussion by JeremieDENAEGEL** [witsa/synapps#719] [Q&A]
Bonjour,

J'ai plusieurs zones regroupants des défauts techniques.
Je souhaite afficher sur la scène principale du synapps le nombre de défauts en cours par zone.

Je sais que cela est possible avec le count d'un acteur "journal des variables", réglé sur les en-cours et avec le filtre de zone qui va bien, pour avoir le nombre de défaut en cours dans une zone.

Mais créer un journal des variables dédié pour chaque zone, ça peut faire beaucoup et alourdir le synapps .

Existe-t'il une autre méthode ?

**Comment by WITNicolas** [comment]
Salut, 
A mon sens pas d'autres methodes :(
Le nombre d'even est justement calculé dynamiquement par le journal en fonction des filtres.
